index.src.html

```diff
@@ -655,6 +655,28 @@ <h2 id="integration-with-html">Integration with HTML</h2>
   with W3C HTML5, those would need to be copied from [[HTML]].
 </section>
 
+<section class="informative">
+  <h2 id="integration-with-css">Integration with CSS</h2>
+
+  The CSS Standard does not specify how it fetches resources referenced from
+  stylesheets. However, implementations use the URL a given stylesheet was
+  loaded from as referrer for the requests for resources from that stylesheet.
+
+  Implementations should keep track of a referrer policy for each stylesheet
+  that should be used to create requests for resources from the respective
+  stylesheet. 
+
+  For external stylesheets, the referrer policy should be
+  <a>"<code>no-referrer-when-downgrade</code>"</a> unless overwritten by an
+  `<code>Referrer-Policy</code>` header.
+
+  For inline stylesheets, the referrer policy is the containing {{Document}}'s
+  referrer policy.
+
+  Both the value of the referrer and the value of the referrer policy should be
+  captured at the time a given request is created.
```

Comment:
Is this really the intention? Or is it at the time the stylesheet is created?

I think this only matters for inline style sheets where the containing document's referrer policy can change over time as meta elements are added/removed.